pkgs/by-name/sa/samrewritten/package.nix

```
@@ -35,6 +34,11 @@ stdenv.mkDerivation (finalAttrs: {
     yajl
   ];
 
+  postInstall = ''
+    substituteInPlace $out/share/applications/samrewritten.desktop \
+      --replace "Exec=/usr/bin/samrewritten" "Exec=$out/bin/samrewritten"
```

```suggestion
      --replace-fail "Exec=/usr/bin/samrewritten" "Exec=samrewritten"
```
see https://github.com/NixOS/nixpkgs/issues/308324